### ISSUE TITLE
Use media queries to discover if a user can hover

### ISSUE THREAD

**Issue by benyakirten** [benyakirten/chat-app#41]
For the buttons regarding an individual chat (such as edit/delete), on a client with hover capabilities (i.e. desktop), the user should have to hover to reveal the buttons. However, for accessibility and mobile devices (i.e. ones with no cursor or the user tabbing in, probably using `:focuswithin` or whatever the pseudo class is), they should be revealed automatically.
